llvm/test/tools/llvm-dwp/X86/empty_warning.test

```diff
@@ -0,0 +1,9 @@
+; RUN: rm -rf %t && mkdir -p %t && split-file %s %t && cd %t
+; RUN: clang -g -gdwarf-4 %t/non_split.c -o %t/non_split.o
```

Comment:
We can't run clang as part of llvm's tests (it's a layering violation - clang depends on llvm, not the other way around).

This test should contain something more basic - perhaps an empty or trivial assembly file that `llvm-mc -g` could be used to compile to an object file and that might be enough for the situation?

Reply:
oh didn't know that.
Why did the test pass? Thought there would be an error "clang not found" or something like that.

Reply:
lit RUN lines basically boil down to a shell script, so it inherits the system PATH - but if you ran the tests on a machine that didn't have clang installed (and didn't build clang) the test would fail, so far as I understand.